src/site/stages/build/drupal/api.js

```diff
@@ -124,12 +190,15 @@ function getDrupalClient(buildOptions) {
       });
     },
 
-    getExportedPages() {
-      const exportDir = buildOptions['cms-export-dir'] || contentDir;
-      const entities = readAllNodeNames(exportDir).map(entityDetails =>
-        readEntity(exportDir, ...entityDetails),
+    getExportedPages({ debug } = { debug: true }) {
```

Comment:
This debug defaults to `true` but the others default to false.  Is there a reason for that?

Reply:
Because consistency. :man_facepalming: